### DISCUSSION TITLE
Is it possible to use the Built-in encoding of the pi camera 3 and watch that video stream in the webcontrol?

### DISCUSSION THREAD

**Discussion by gismo1187** [Motion-Project/motionplus#127] [Q&A]
Hey guys
 
I have a pi zero 2 w and a pi cam 3 NoIR. 
OS: Raspberry Pi OS 64bit
Basically i´ve got it working, but the video Stream in the webcontrol  is extremely laggy. Around 1 fps.
In the settings I set it to 30fps. at 1080 resolution.  And because in my usecase I will not use motion detection an not record anything so it would be nice if the stream would be more viewable.

Thanks

**Comment by axeljerabek** [comment]
I run the same setup, without the NoIR though. I easily get 30 fps into a stream, but the motion detection is running on a different machine. I transport the 30 fps via flv stream to an RTMP Nginx and record there with motion. I think that the motion detection is taking too much horepower from the CPU.
